src/test/java/org/broadinstitute/consent/http/db/DAOTestHelper.java

```diff
@@ -241,7 +241,7 @@ protected Vote createPopulatedChairpersonVote(Integer userId, Integer electionId
     }
 
     @SuppressWarnings("SameParameterValue")
-    protected Consent createConsent(Integer dacId) {
+    protected Consent createConsent() {
```

Comment:
Commenting here because I can't on line 163. In this class, we might need to re-arrange the deletion order a bit since datasets have a dac id and we're deleting datasets before dacs.

Reply:
Looks like we're still deleting datasets before dacs (lines 166, 168). Tests are passing now, but it could be a problem if we create a test that associates a dac to a dataset, then try to delete that dataset before deleting the dac.